packages/lib/server/queries/teams/index.ts

```diff
@@ -136,33 +131,36 @@ export async function getTeamWithMembers(args: {
   // This should improve performance saving already app data found.
   const appDataMap = new Map();
   const members = team.members.map((obj) => {
+    const { credentials, ...restUser } = obj.user;
     return {
-      ...obj.user,
+      ...restUser,
```

Comment:
Credentials were still added to the returned object for each member even when its only use was to determine "connectedApps" at the bottom 🤦